openassessment/workflow/models.py

```diff
@@ -346,12 +355,61 @@ def _get_steps(self):
             # If no steps exist for this AssessmentWorkflow, assume
             # peer -> self for backwards compatibility
             self.steps.add(
-                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=0),
-                AssessmentWorkflowStep(name=self.STATUS.self, order_num=1)
+                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=1),
+                AssessmentWorkflowStep(name=self.STATUS.self, order_num=2)
             )
             steps = list(self.steps.all())
+
+        # A staff step will always be available, to allow for staff overrides
```

Comment:
I'm not sure it's possible for a staff assessment to exist without this step being available? I tried just adding a `self.staff_score_exists()` check here, but that quickly blows up the stack due to recursion, since that method calls into `self._get_steps()`